packages/teams-js/src/public/authentication.ts

```diff
@@ -668,6 +668,34 @@ export namespace authentication {
      * Limited to Microsoft-internal use
      */
     ver: string;
+    /**
+     * @hidden
+     * Stores the residency of the user.
+     *
+     * @internal
+     * Limited to Microsoft-internal use
+     */
+    dataResidency: DataResidencySlim;
+  }
+
+  /**
+   * Limited set of data residencies information exposed to 1P application developers
+   */
+  export enum DataResidencySlim {
+    /**
+     * Public
+     */
+    Public = 'public',
+
+    /**
+     * European Union Data Boundary
+     */
+    EUDB = 'eudb',
+
+    /**
+     * Other, stored to cover fields that will not be exposed
+     */
+    Other = 'other',
```

Comment:
Is this the value that's returned if the host doesn't provide this property when `getUser` is called? Would it be valuable to have a separate enum value to distinguish between "the host gave us a data residency value but it's not public/eudb" and "the host didn't give us a value." ?

Reply:
I think because `dataResidency` is optional on `UserProfile` that's already covered by not having it there at all?

Reply:
Yes, that would do it.